public/v4/apps/minio.yml

```diff
@@ -28,19 +27,19 @@ caproverOneClickApp:
     variables:
         - id: $$cap_minio_version
           label: MinIO Version
-          defaultValue: RELEASE.2023-01-18T04-36-38Z
+          defaultValue: RELEASE.2025-09-07T16-13-09Z
           description: Check out their Docker page for the valid tags https://hub.docker.com/r/minio/minio/tags/
           validRegex: /^([^\s^\/])+$/
         - id: $$cap_access_key
           label: MinIO Root User Access Key
-          defaultValue: $$cap_gen_random_hex(24)
+          defaultValue: admin
           description: Access key for `MINIO_ROOT_USER`. If unset, minio defaults to `minioadmin`. MinIO strongly recommends specifying a unique, long, and random value for all environments.
-          validRegex: /(.{5,})|(^\s{0}$)/m
+          validRegex: /^(.{5,})?$/
         - id: $$cap_secret_key
           label: Minio Secret Key
-          defaultValue: $$cap_gen_random_hex(38)
-          description: Secret key for `MINIO_ROOT_PASSWORD`. If unset, minio defaults to `minioadmin`. MinIO strongly recommends specifying a unique, long, and random value for all environments.
-          validRegex: /(.{8,})|(^\s{0}$)/m
+          defaultValue: password
```

Comment:
Should be kept unchanged ` $$cap_gen_random_hex(38)` ensures random default value